src/store/app/state.ts

```diff
@@ -1,4 +1,9 @@
 // eslint-disable-next-line @typescript-eslint/ban-types
-export type S = {}
+export type S = {
+  /** 検索履歴 新しい順に並ぶ */
+  searchHistories: string[] //TODO 型
```

Comment:
```suggestion
  messageSearchHistories: string[]
```
検索はメッセージ検索以外にも存在するので(ほとんどはfilterって言い方にはなっていますが)一応`message`をつけておきたいです :pray:
あと型はこれで問題ないです